### ISSUE TITLE
Explode / Compress Box sets

### ISSUE THREAD

**Issue by Omertron** [YAMJ/yamj-v2#1724]
Original [issue 1725](https://code.google.com/p/moviejukebox/issues/detail?id=1725) created by Omertron on 2010-11-26T16:55:45.000Z:

Using Explodesets it would be nice to have better fine tuning as to which categories this can be applied to.... i.e I know a can put Other in the properties to have every Other category exploded.  put it would be nice to be able to list specific &quot;other&quot; categories by name...like other_new or other_all....

Similarly:
After the addition of the &quot;expand sets for certain categories&quot; I'm not sure how to list the categories to work as such:

I've never liked the fact that for Tomorrow never dies, it shows up in the &quot;T&quot;'s...for me I've a boxset defined for &quot;James Bond&quot; which i'd expect to show under &quot;J&quot;s and not have ANY james bond movies show in any other TITLE category.

In other words the I'd like the TITLE pages to look just like the OTHER pages, but I'm not sure if this setting allows me to do this. I think what I might be looking for is a COMPRESS BOX Sets so that A movie can only be accessed through the set that defines it.
